CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changes
 =======
 
+# 0.3.1 / 11-03-2015
```

Comment:
Nvm, solved in this cherry-pick PR: https://github.com/DataDog/jenkins-datadog-plugin/pull/29